### PR TITLE
Selenis part1 exercise

### PR DESCRIPTION
**Pull request by SabineSt**
couldnt make #7 work

### CODE REVIEW

File: src/main/java/com/codingnomads/part_01/Exercise_02.java [hunk 1]
```diff
@@ -6,5 +6,14 @@
  */
 
 public class Exercise_02 {
+    public static void main(String[] args) {
+        String str = "Hello World!";
+
+        System.out.println(str);
```
Comment (roioteromorales) [inline src/main/java/com/codingnomads/part_01/Exercise_02.java]: There is a mantra in programming, its called DRY, https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Any other way to do this without duplication?

File: src/main/java/com/codingnomads/part_01/Exercise_04.java [hunk 2]
```diff
@@ -11,5 +11,16 @@
  */
 
 public class Exercise_04 {
+    public static void main(String[] args) {
+        System.out.println("a\ta*2\ta*4");
 
+        for(int i = 1; i<5; i++){
+            for(int t = 1; t <4; t++){
```
Comment (roioteromorales) [inline src/main/java/com/codingnomads/part_01/Exercise_04.java]: this code is working but is overcomplicated , hard to read and the performance is not really good.
Try to think about simpler solutions:
```
        System.out.println("a\ta*2\ta*4");

        for (int i = 1; i <= 4; i++) {
            String calculation = "";
            for (int t = 1; t <= 4; t = t * 2) {
                calculation += i * t + "\t";
            }
            System.out.println(calculation);
        }
```

This solution aswell allows you to extend it to more rows and columns. lets say` i < 100 `or `t < 100 `

File: src/main/java/com/codingnomads/part_01/Exercise_08.java [hunk 3]
```diff
@@ -9,4 +10,18 @@
 
 public class Exercise_08 {
 
+    public static void main(String[] args) {
+        double radius = 3.14;
+        double area;
+        double perimeter;
+
+        perimeter = 2* Math.PI * radius;
+        area = (perimeter*perimeter)/4* Math.PI;
```
Comment (roioteromorales) [inline src/main/java/com/codingnomads/part_01/Exercise_08.java]: This is not the correct formula for calculating the area of a circle.

File: src/main/java/com/codingnomads/part_01/Exercise_09.java [hunk 4]
```diff
@@ -7,6 +7,31 @@
  * a rectangle that has a width of 2.4 and a height of 6.4
  */
 
+ class Rectangle {
```
Comment (roioteromorales) [inline src/main/java/com/codingnomads/part_01/Exercise_09.java]: Nice! you even used a class!! awesome!

File: src/main/java/com/codingnomads/part_01/Exercise_10.java [hunk 5]
```diff
@@ -8,5 +8,14 @@
  */
 
 public class Exercise_10 {
+    public static void main(String[] args) {
+        double route = 12/1.6;
+        double speed = 30.00 / 60.00 + 30.00/3600;
+        double averageSpeed = route/speed;
+
+       // System.out.println(route);
```
Comment (roioteromorales) [inline src/main/java/com/codingnomads/part_01/Exercise_10.java]: Try to never commit commented out code!

File: src/main/java/com/codingnomads/part_01/Exercise_10.java [hunk 6]
```diff
@@ -8,5 +8,14 @@
  */
 
 public class Exercise_10 {
+    public static void main(String[] args) {
+        double route = 12/1.6;
```
Comment (roioteromorales) [inline src/main/java/com/codingnomads/part_01/Exercise_10.java]: Try to use more meaningful variables names, like `distanceInMiles`

File: src/main/java/com/codingnomads/part_01/Exercise_02.java (hunk 1 above)
Comment (SabineSt) [reply src/main/java/com/codingnomads/part_01/Exercise_02.java]: yessah! for loop

File: src/main/java/com/codingnomads/part_01/Exercise_08.java (hunk 3 above)
Comment (SabineSt) [reply src/main/java/com/codingnomads/part_01/Exercise_08.java]: ok! fixed it

File: src/main/java/com/codingnomads/part_01/Exercise_09.java (hunk 4 above)
Comment (SabineSt) [reply src/main/java/com/codingnomads/part_01/Exercise_09.java]: thanx! :)

File: src/main/java/com/codingnomads/part_01/Exercise_10.java (hunk 6 above)
Comment (SabineSt) [reply src/main/java/com/codingnomads/part_01/Exercise_10.java]: ok. changed it

File: src/main/java/com/codingnomads/part_01/Exercise_10.java (hunk 5 above)
Comment (SabineSt) [reply src/main/java/com/codingnomads/part_01/Exercise_10.java]: ok.

File: src/main/java/com/codingnomads/part_01/Exercise_04.java (hunk 2 above)
Comment (SabineSt) [reply src/main/java/com/codingnomads/part_01/Exercise_04.java]: ok! adapted it.

**Comment by roioteromorales** [comment]
Awesome ! nice job!

**Pull request merged**